### ISSUE TITLE
Stuck at waiting for server to start up

### ISSUE THREAD

**Issue by chankeypathak** [brianmed/liveperl.us#16]
http://liveperl.us/tutorial/hello does not work any more

**Comment by brianmed** [comment]
I made a change, will you try again please?

**Comment by chankeypathak** [comment]
No, it's still not working. You can have a look at the mentioned URL. It gets stuck at "Waiting for server to start up...".

**Comment by brianmed** [comment]
Will you let me know when you can try again, and I can watch the logs on the server?

**Comment by chankeypathak** [comment]
This site can’t be reached

liveperl.us’s server DNS address could not be found.

Tried at 1:07 PM (IST)
